src/app/components/Game/components/Board/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Square from './components/Square';
+import styles from './styles.scss';
+
+class Board extends Component {
+  renderSquare(i) {
+    return <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
+  }
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
import Fragment on the first line like `{ Component, Fragment }` and use just `<Fragment>` here